src/main/java/Storage.java

```diff
@@ -0,0 +1,32 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Storage {
+    File file;
+
+    public Storage (String filePath) {
+        File file = new File(filePath);
```

Comment:
```suggestion
        this.file = new File(filePath);
```
'File file' creates a local variable